comment-updater-plugin/src/main/resources/inspectionDescriptions/CodeComment.html

```diff
@@ -0,0 +1,6 @@
+<html>
```

Comment:
why do we need it?

Reply:
In IDEA you can click on warning "Inconsistency detected" and click three dots button -> view inspection description. This file specifies text of this inspection description.